docs/concepts/configuration/assign-pod-node.md

```diff
@@ -132,7 +132,7 @@ This node affinity rule says the pod can only be placed on a node with a label w
 among nodes that meet that criteria, nodes with a label whose key is `another-node-label-key` and whose
 value is `another-node-label-value` should be preferred.
 
-You can see the operator `In` being used in the example. The new node affinity syntax supports the following operators: `In`, `NotIn`, `Exists`, `DoesNotExist`, `Gt`, `Lt`.
+You can see the operator `In` being used in the example. The new node affinity syntax supports the following operators: `In`, `NotIn`, `Exists`, `DoesNotExist`.
```

Comment:
Node affinity works with all 6 operators: `In`, `NotIn`, `Exists`, `DoesNotExist`, `Gt`, `Lt` . pod affinity/anti-affinity only works with regular 4 operators: `In`, `NotIn`, `Exists`, `DoesNotExist`.